app/pages/sections/catalogue-solutions/order-item/controller.test.js

```diff
@@ -19,6 +19,15 @@ jest.mock('./manifestProvider', () => ({
   getSelectedPriceManifest: jest.fn(),
 }));
 
+const serviceRecipient = {
+  name: 'Recipient 1',
+  odsCode: 'ods1',
+};
+const solution = {
+  id: 'solutionId1',
+  name: 'Solution 1',
+};
+
```

Comment:
same with the above 2 consts if you could move them to the test that uses them. I think it's only being used by that one test